Sources/ArgumentParser/Documentation.docc/Articles/CustomizingHelp.md

```diff
@@ -150,6 +150,150 @@ OPTIONS:
 
 For an ``ExpressibleByArgument`` and `CaseIterable` type with many cases, you may still want to implement ``ExpressibleByArgument/allValueStrings`` to avoid an overly long list of values appearing in the help screen. For these types it is recommended to include the most common possible values.
 
+## Providing Descriptions for Individual Enum Values
+
+When your argument or option uses an enum type, you can provide detailed descriptions for each enum value that will appear in the help screen. This is especially useful when the enum cases represent complex concepts that benefit from explanation.
+
+### Basic Enum Value Descriptions
```

Comment:
can we use just one enum throughout this documentation, I think output format is a good example